R/model-modules.R

```diff
@@ -326,20 +330,21 @@ enw_report <- function(non_parametric = ~0, structural = ~0, data) {
 #' @family modelmodules
 #' @importFrom rstan extract_sparse_parts
 #' @importFrom purrr map2_dbl
+#' @importFrom rlang abort
 #' @export
 #' @examples
 #' enw_expectation(data = enw_example("preprocessed"))
 enw_expectation <- function(r = ~ 0 + (1 | day:.group), generation_time = 1,
                             observation = ~1, latent_reporting_delay = 1,
                             data, ...) {
   if (as_string_formula(r) %in% "~0") {
-    stop("An expectation model formula for r must be specified")
+    rlang::abort("An expectation model formula for r must be specified")
   }
   if (as_string_formula(observation) %in% "~0") {
     observation <- ~1
   }
   if (sum(generation_time) != 1) {
-    stop("The generation time must sum to 1")
+    rlang::abort("The generation time must sum to 1")
```

Comment:
The `generation_time` pmf ? ...or similar?

Reply:
yes but not for this PR